src/lib/services/project-service.ts

```diff
@@ -596,20 +617,61 @@ export default class ProjectService {
     async statusJob(): Promise<void> {
         const projects = await this.store.getAll();
 
+        const statusUpdates = await Promise.all(
+            projects.map((project) => this.getStatusUpdates(project.id)),
+        );
+
         await Promise.all(
-            projects.map((project) =>
-                this.calculateAverageTimeToProd(project.id),
-            ),
+            statusUpdates.map((statusUpdate) => {
+                return this.projectStatusStore.updateStatus(
+                    statusUpdate.projectId,
+                    statusUpdate.updates,
+                );
+            }),
         );
     }
 
-    async calculateAverageTimeToProd(projectId: string): Promise<number> {
+    async getStatusUpdates(projectId: string): Promise<ICalculateStatus> {
         // Get all features for project with type release
         const features = await this.featureToggleStore.getAll({
             type: 'release',
             project: projectId,
         });
 
+        const [createdCurrentWindow, createdPastWindow] = await Promise.all([
+            await this.featureToggleStore.getByDate({
+                project: projectId,
+                dateAccessor: 'created_at',
+                date: subDays(new Date(), 30).toISOString(),
+            }),
+            await this.featureToggleStore.getByDate({
+                project: projectId,
+                dateAccessor: 'created_at',
+                range: [
+                    subDays(new Date(), 60).toISOString(),
```

Comment:
what about intention revealing local variable name for the -60 days and -30 days?

Reply:
Good idea.